### ISSUE TITLE
Wrong icon since kf5.12

### ISSUE THREAD

**Issue by FedeDP** [kotelnik/plasma-applet-weather-widget#4]
Hi!
I just upgraded to kf5.12 and i noticed i have no icon (just a white square).
Probably the bug was caused by: 
"KIconThemes
Don't try to paint the icon if the size is invalid" (https://www.kde.org/announcements/kde-frameworks-5.12.0.php).

See screenshot: http://postimg.org/image/3liotophf/ .

**Comment by kotelnik** [comment]
Hello! Thanks for report. I just updated and I can see it, too. And it affects other widgets... It looks like there is a problem with loading in-plasmoid resources - e.g. loading font in path "plasmapackage:/path/to/font.ttf" is outputing to log that protocol plasmapackage is unknown. I don't know of any other way to load stuff inside plasma package so unfortunately I cannot do anything about it right now.

I filed a bug though so hopefully it will be explained/fixed soon.
https://bugs.kde.org/show_bug.cgi?id=350101

Workaround for now is to install this font temporarily to personal fonts in Font Management in KDE:
https://github.com/kotelnik/plasma-applet-weather-widget/blob/master/org.kde.weatherWidget/contents/fonts/weathericons-regular-webfont.ttf
